src/Management/src/Endpoint/Info/Contributor/GitInfoContributor.cs

```diff
@@ -28,31 +27,29 @@ public GitInfoContributor(ILogger<GitInfoContributor> logger)
     {
     }
 
-    private GitInfoContributor(string propertiesPath, ILogger<GitInfoContributor> logger)
+    public GitInfoContributor(string propertiesPath, ILogger<GitInfoContributor> logger)
```

Comment:
I take it there wasn't actually a reasonable way to change the path later or use more than one file to contribute git info? Seems we didn't have any tests at least

Reply:
I'm not sure I get your point. So I'm going to assume you're referring to the removal of the `string propertiesPath` parameter from the `ReadGitPropertiesAsync` method. In that case: that's correct. The [original code](https://github.com/SteeltoeOSS/Steeltoe/blob/main/src/Management/src/Endpoint/Info/Contributor/GitInfoContributor.cs#L45) always passed the read-only field value:

```c#
Configuration = await ReadGitPropertiesAsync(_propertiesPath, cancellationToken);
```

I suspect the parameter existed to facilitate writing tests, but it's not actually needed, so I removed it. Note the type itself is internal.

In case you meant to raise something else, can you please clarify?

Reply:
Sorry about writing that out poorly, I was rushing a bit on that note. This is an item that's been (very low) on my list to understand a bit better because of how the git info frequently doesn't work out the best from our sample and some other container scenarios, but that is probably unrelated